circuitpython_library_pypi_stats.md

```diff
@@ -3,6 +3,10 @@
 Here is a listing of current Adafruit CircuitPython libraries download statistics.  
 There are 308 libraries available.
 
+
+Total Blinka downloads: 34218  
+Total PyPI library downloads: 182737
+
 | PyPI Package | Downloads in the Last 7 Days |
```

Comment:
I feel like we should do it as follows.
```
| Library (PyPI Package) | Downloads in the Last 7 Days |
```
I will update one listing below to explain more clearly.